kontext - bucket manager doesn't load images

```
2025-07-17 19:44:56,581 [INFO] Configured backend: {'id': 'my-edited-images', 'config': {'crop': False, 'crop_aspect': 'square', 'crop_style': 'random', 'disable_validation': False, 'resolution': 1.048576, 'resolution_type': 'area', 'caption_strategy': 'textfile', 'instance_data_dir': 'datasets/edited-images', 'maximum_image_size': None, 'target_downsample_size': None, 'dataset_type': 'image', 'config_version': 2}, 'dataset_type': 'image', 'instance_data_dir': 'datasets/edited-images'}
(Rank: 0)  | bucket     | image count (per-GPU)
------------------------------
(Rank: 0)  | 1.0        | 8
(Rank: 0)  | 0.65       | 1
No images were discovered by the bucket manager in the dataset: my-edited-images.
Traceback (most recent call last):
  File "/workflow/SimpleTuner2/train.py", line 50, in <module>
    trainer.init_data_backend()
  File "/workflow/SimpleTuner2/helpers/training/trainer.py", line 512, in init_data_backend
    raise e
  File "/workflow/SimpleTuner2/helpers/training/trainer.py", line 485, in init_data_backend
    configure_multi_databackend(
  File "/workflow/SimpleTuner2/helpers/data_backend/factory.py", line 1411, in configure_multi_databackend
    raise Exception(
Exception: No images were discovered by the bucket manager in the dataset: my-edited-images.
2025-07-17 19:44:56,585 [INFO] (id=my-edited-images) Collecting captions.
2025-07-17 19:44:56,587 [INFO] (id=my-edited-images) Initialise text embed pre-computation using the textfile caption strategy. We have 16 captions to process.
2025-07-17 19:44:56,874 [INFO] (id=my-edited-images) Completed processing 16 captions.
2025-07-17 19:44:56,874 [INFO] (id=my-edited-images) Creating VAE latent cache.
2025-07-17 19:44:56,876 [ERROR] VAE image embed cache directory None is not set. This is required for the VAE image embed cache., traceback: Traceback (most recent call last):
  File "/workflow/SimpleTuner2/helpers/training/trainer.py", line 485, in init_data_backend
    configure_multi_databackend(
  File "/workflow/SimpleTuner2/helpers/data_backend/factory.py", line 1612, in configure_multi_databackend
    raise ValueError(
ValueError: VAE image embed cache directory None is not set. This is required for the VAE image embed cache.
VAE image embed cache directory None is not set. This is required for the VAE image embed cache.
Traceback (most recent call last):
  File "/workflow/SimpleTuner2/train.py", line 50, in <module>
    trainer.init_data_backend()
  File "/workflow/SimpleTuner2/helpers/training/trainer.py", line 512, in init_data_backend
    raise e
  File "/workflow/SimpleTuner2/helpers/training/trainer.py", line 485, in init_data_backend
    configure_multi_databackend(
  File "/workflow/SimpleTuner2/helpers/data_backend/factory.py", line 1612, in configure_multi_databackend
    raise ValueError(
ValueError: VAE image embed cache directory None is not set. This is required for the VAE image embed cache.
[rank0]:[W717 19:44:58.342193611 ProcessGroupNCCL.cpp:1476] Warning: WARNING: destroy_process_group() was not called before program exit, which can leak resources. For more info, please see https://pytorch.org/docs/stable/distributed.html#shutdown (function operator())
```

--- a/documentation/quickstart/FLUX_KONTEXT.md
+++ b/documentation/quickstart/FLUX_KONTEXT.md
@@ -74,7 +74,8 @@ The `conditioning_data` field in the edit dataset should point to the reference 
   {
     "id": "my-edited-images",
     "type": "local",
-    "instance_data_dir": "/path/to/datasets/edited-images",     // <-- use absolute paths
+    "cache_dir_vae": "/cache/vae/flux/kontext/edited-images",   // <-- where VAE outputs are stored
+    "instance_data_dir": "/datasets/edited-images",             // <-- use absolute paths
     "conditioning_data": [
       "my-reference-images"                                     // <‑‑ this should be your "id" of the reference set
                                                                 // you could specify a second set to alternate between or combine them, e.g. ["reference-images", "reference-images2"]
@@ -85,7 +86,8 @@ The `conditioning_data` field in the edit dataset should point to the reference 
   {
     "id": "my-reference-images",
     "type": "local",
-    "instance_data_dir": "/path/to/datasets/reference-images",  // <-- use absolute paths
+    "cache_dir_vae": "/cache/vae/flux/kontext/ref-images",      // <-- where VAE outputs are stored. must be different from other dataset VAE paths.
+    "instance_data_dir": "/datasets/reference-images",          // <-- use absolute paths
     "conditioning_type": "reference_strict",                    // <‑‑ if this is set to reference_loose, the images are cropped independently of the edit images
     "resolution": 1024,
     "caption_strategy": null,                                   // <‑‑ no captions needed for references, but if available, will be used INSTEAD of the edit captions

